opengrok-indexer/src/main/java/org/opengrok/indexer/history/HistoryGuru.java

```diff
@@ -731,24 +729,16 @@ public void ensureHistoryCacheExists(File file) throws HistoryException {
 
         Repository repository = getRepository(file);
         if (repository == null) {
-            // no repository -> no history :(
```

Comment:
:(

Reply:
depends on point of view I guess. Also, emojis are now the cool thing to have in (UTF-8 encoded) source code.